src/Sylius/Behat/Page/Admin/ProductAttribute/UpdatePage.php

```diff
@@ -68,18 +69,25 @@ public function hasAttributeValue(string $value): bool
     /**
      * {@inheritdoc}
      */
-    public function addAttributeValue(string $value): void
+    public function addAttributeValue(string $value, string $localeCode): void
     {
         $this->getDocument()->clickLink('Add');
-        $this->getLastAttributeChoiceElement()->find('css', 'input')->setValue($value);
+        $this
+            ->getLastAttributeChoiceElement()
+            ->find('css', 'div[data-locale="' . $localeCode . '"] input')
+            ->setValue($value)
+        ;
     }
 
     /**
      * {@inheritdoc}
      */
     public function deleteAttributeValue(string $value): void
     {
-        $attributeChoiceElement = $this->getElement('attribute_choice_list_element', ['%value%' => $value])->getParent();
+        $attributeChoiceElement = $this
+            ->getElement('attribute_choice_list_element', ['%value%' => $value])
+            ->getParent()->getParent()->getParent()->getParent()
```

Comment:
Hmm, is there any way we can get rid of those parent calls?

Reply:
I've been trying to come up with better solution, but without success. If you have any suggestion, please share with me 😃